pulsar-flink/src/test/scala/org/apache/flink/batch/connectors/pulsar/example/README.md

```diff
@@ -0,0 +1,262 @@
+The Flink Batch Sink for Pulsar is a custom sink that enables Apache [Flink](https://flink.apache.org/) to write [DataSet](https://ci.apache.org/projects/flink/flink-docs-stable/dev/batch/index.html) to Pulsar.
```

Comment:
Can you move the examples to `examples` module? e.g. https://github.com/apache/pulsar/tree/master/examples/flink-consumer-source

Reply:
@sijie Both `Java` and `Scala` Flink Examples have been moved under `/examples/flink-consumer-source`
I think, as current patch, `single Flink example sub-module` can cover all Flink related examples.
On the other hand, Batch and Streaming Documentations can also be aligned. Plan to address via separated PR if there is no concern.